src/galaxy/SystemBody.h

```diff
@@ -271,6 +271,8 @@ class SystemBody : public RefCounted, public SystemBodyType, protected SystemBod
 		// which is rendered when the body has a small screen size
 		return Color(200, 200, 200, 255);
 	}
+	double GetPlanckBrightness(const double wavelength_nm, const int temperature) const;
+	Color GetColor() const;
```

Comment:
I'd prefer if this was more explicitly named as it's only for stars, e.g. `GetStarColor()`.